brands/amenity/cafe.json

```diff
@@ -113,6 +113,19 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|BreadTalk": {
+    "countryCodes": ["sg"],
+    "matchNames": ["Bread Talk"],
```

Comment:
This `matchName` is unnecessary and can be removed - the matching code ignores capitalization and spaces..